plutus-wallet-api/ledger/Ledger/Tx.hs

```diff
@@ -140,6 +142,19 @@ data Tx = Tx {
     } deriving stock (Show, Eq, Generic)
       deriving anyclass (ToJSON, FromJSON, Serialise)
 
+instance Semigroup Tx where
```

Comment:
we also need an instance for `Language.Plutus.Contract.Tx.UnbalancedTx`

Reply:
:+1: